### PR TITLE
Add some article boilerplate

### PR DESCRIPTION
**Pull request by jtrussell**
Just prepping for when we have article content to include.

Screen Shot 2019-09-17 at 10 46 31 PM

Screen Shot 2019-09-17 at 10 46 40 PM

### CODE REVIEW

File: next/components/article.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import { useContext } from 'react';
+import { ThemeContext } from './theme-context';
+
+function Article({ title, url, description, date }) {
+  const theme = useContext(ThemeContext);
+
+  const now = new Date();
```
Comment (evsheffield) [inline next/components/article.js]: We should be able to leverage the existing time util for this: https://github.com/mythgard-hub/mythgard-hub/blob/801ef8a57e078151e2b8ec4960484e52fca7b699/next/lib/time.js

File: next/components/article.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      </div>
+    </>
+  );
+}
+
+Article.propTypes = {
+  title: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  date: PropTypes.instanceOf(Date)
```
Comment (evsheffield) [inline next/components/article.js]: Should this one also be required? Seems like there may be sad times with `date.getTime()` if this prop isn't provided.

File: next/components/article.js (hunk 1 above)
Comment (jtrussell) [reply next/components/article.js]: Oh nice, I didn't know this existed 😅. Looks very familiar ;) https://github.com/mythgard-hub/mythgard-hub/pull/68/files#diff-55725476055ab916956f3bdf9b2cd93aR8-R26

File: next/components/article.js (hunk 2 above)
Comment (jtrussell) [reply next/components/article.js]: Yup, sure should.

**Pull request merged**